homeassistant/components/airzone/__init__.py

```diff
@@ -70,7 +72,11 @@ def _async_migrator(entity_entry: er.RegistryEntry) -> dict[str, Any] | None:
         }
         hass.config_entries.async_update_entry(entry, **updates)
```

Comment:
Do we need to check for an empty string as mac address in the config entry and entity unique id migration to mac address above? If that could happen it would take a reload before it's fixed for the config entry with the current code and the entity entries would still be incorrect.